ext/js/display/display-anki.js

```diff
@@ -1121,6 +1122,20 @@ export class DisplayAnki {
         viewNoteButton.hidden = disabled;
         viewNoteButton.dataset.noteIds = noteIds.join(' ');
 
+        let isSuspended = false;
+        for (const item of noteInfos) {
+            if (item === null) { continue; }
+            for (const cardInfo of item.cardsInfo) {
+                if (cardInfo.queue === -1) {
+                    isSuspended = true;
+                    break;
+                }
+            }
+        }
+        if (isSuspended) {
+            viewNoteButton.style.setProperty('background-color', 'yellow', 'important');
```

Comment:
This should be a class defined in the popup css.

Reply:
I wasn't sure how it get it to work in `action-popup.css` so I put it in `display.css` as that is where the other css classes were for the buttons